engine/src/main/java/org/terasology/engine/network/internal/NetworkSystemImpl.java

```diff
@@ -855,7 +856,7 @@ private void processNewClient(NetClient client) {
         connectClient(client);
 
         // log after connect so that the name has been set:
-        logger.info("New client entity: {}", client.getEntity());
+        logger.atInfo().addArgument(() -> client.getEntity()).log("New client entity: {}");
```

Comment:
info-level log and only called when client added so impact neglectable
```suggestion
        logger.info("New client entity: {}", client.getEntity()); //NOPMD
```